src/garage/tf/optimizers/conjugate_gradient_optimizer.py

```diff
@@ -192,29 +305,34 @@ def __init__(self,
             hvp_approach = PerlmutterHvp(num_slices)
         self._hvp_approach = hvp_approach
 
-    def update_opt(self,
-                   loss,
-                   target,
-                   leq_constraint,
-                   inputs,
-                   extra_inputs=None,
-                   name=None,
-                   constraint_name='constraint',
-                   *args,
-                   **kwargs):
-        """
-        :param loss: Symbolic expression for the loss function.
-        :param target: A parameterized object to optimize over. It should
-         implement methods of the
-         the :class:`garage.core.paramerized.Parameterized` class.
-        :param leq_constraint: A constraint provided as a tuple (f, epsilon),
-         of the form f(*inputs) <= epsilon.
-        :param inputs: A list of symbolic variables as inputs, which could be
-         subsampled if needed. It is assumed that the first dimension of these
-         inputs should correspond to the number of data points
-        :param extra_inputs: A list of symbolic variables as extra inputs which
-         should not be subsampled
-        :return: No return value.
+    def update_opt(
+            self,
+            loss,
+            target,
+            leq_constraint,
+            inputs,
+            extra_inputs=None,
+            name=None,
+            constraint_name='constraint',
+    ):
+        """Update class properties.
+
+        Args:
+            loss (tf.Tensor): Symbolic expression for the loss function.
+            target (garage.tf.policies.Policy): A parameterized object to
+                optimize over.
+            leq_constraint (tuple(tf.Tensor, float)): A constraint provided
```

Comment:
collections usually use [] for type annotations (e.g. `tuple[tf.Tensor, float]`, `list[tf.Tensor]`)